ivadomed/losses.py

```diff
@@ -21,7 +21,7 @@ def forward(self, prediction, target):
         for i in self.classes_of_interest:
             dice_per_class += dice_loss(prediction[:, i, ], target[:, i, ])
 
-        return dice_per_class / len(self.classes_of_interest)
+        return - dice_per_class / len(self.classes_of_interest)
```

Comment:
Would it be okay to keep it positive in the return?
Can't remember exactly why but we decided this with @olix86 last Autumn --> we can re-disucss of course if needed :)
I think it was a "convention" across the loss, so that they can be combined knowing they are positive.

Reply:
This would require that I add a `if` in the code (if multi_class_dice), because the other loss functions don't require a negative